ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/AddMissingSeqFixTest.fs

```diff
@@ -0,0 +1,18 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Intentions.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+[<AssertCorrectTreeStructure>]
+type AddMissingSeqFixTest() =
+    inherit FSharpQuickFixTestBase<AddMissingSeqFix>()
+
+    override x.RelativeTestDataPath = "features/quickFixes/addMissingSeqFix"
+
+    [<Test>] member x.``FS0740 Adds missing seq before ce sequential`` () = x.DoNamedTest()
+    [<Test>] member x.``FS0740 Adds missing seq before foreach sequential`` () = x.DoNamedTest()
+    [<Test>] member x.``FS0740 Adds missing seq before let ce sequential`` () = x.DoNamedTest()
+    [<Test>] member x.``FS0740 Adds parens when needed app`` () = x.DoNamedTest()
+    [<Test>] member x.``FS0740 Adds parens when needed dot`` () = x.DoNamedTest()
+    [<Test>] member x.``FS0740 Adds parens when needed multiline`` () = x.DoNamedTest()
```

Comment:
@edgarfgp Could you add at least one test for the error as well, please?

Reply:
Added one test for FS3873

Reply:
Please try setting the language version via the attribute, like in https://github.com/JetBrains/resharper-fsharp/blob/580d49d3d45c01f6a5869e7988586cbced456bc4/ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/Daemon/IdentifierHighlightingTest.fs#L27-L28